modDesc.xml

```diff
@@ -8,12 +8,14 @@
         <en>Seasons</en>
         <fr>Seasons</fr>
         <de>Seasons</de>
+        <es>Estaciones</es>
```

Comment:
Just when you refer to the mod, and not actual seasons.

'Seasons is a mod that...' <-- keep Seasons
'... implements seasons' <-- translate